third_party/java/arcs/build_defs/internal/kotlin.bzl

```diff
@@ -445,6 +451,7 @@ def arcs_kt_plan(name, srcs = [], data = [], deps = [], platforms = ["jvm"], vis
       data: list of Arcs manifests needed at runtime
```

Comment:
eventually we should convert these to skylark rules so we can use providers to roll up the transitive deps and avoid suffix magic and data vs deps tricks.

Reply:
I totally agree. For now, I didn't want to break any interfaces.